### DISCUSSION TITLE
Pyscript timezone seems to be UTC instead of local

### DISCUSSION THREAD

**Discussion by ryaowe** [custom-components/pyscript#574] [Q&A]
I just migrated my Home Assistant to a new machine (Raspberry Pi 5, core 2024.1.3, supervisor 2023.12.0, OS 11.4, pyscript 1.5.0), and for some reason all of my pyscript `@time_trigger` functions seem to trigger on UTC time instead of local time. Has anyone else seen this before?

`Settings -> System -> General` shows the correct time zone.  `{{ now() }}` in the template dev tool shows local time. But if I run `log.warning(datetime.datetime.now())` in pyscript, it shows UTC time.

If I ssh in and run `date` it also shows the correct time zone and local time. So it seems to be just pyscript that has the wrong zone.

I can't seem to find any configuration option to set the time zone for pyscript. I've tried rebooting the machine, as well as uninstalling and reinstalling pyscript. 

![image](https://github.com/custom-components/pyscript/assets/7231257/586c1e4a-252e-4d9d-94ce-eadccbb356d5)

Any thoughts as to what I might be doing wrong?

**Comment by ALERTua** [comment]
You can use
```
import homeassistant.helpers.template as template

now = template.now(hass)
```
to get the "now" datetime object from HA.

**Comment by ryaowe** [comment]
I nuked the whole setup and installed HA again, and now it seems to work correctly. I think last time I installed HA, it wasn't able to NTP sync at first due to network issues. I'm not sure how that resulted in this weird scenario... but a fresh install seems to have corrected it.
